packages/base/src/sizedMixin.ts

```diff
@@ -17,7 +17,7 @@ type Constructor<T = Record<string, unknown>> = {
     prototype: T;
 };
 
-type ElementSize = 's' | 'm' | 'l' | 'xl';
+export type ElementSize = 's' | 'm' | 'l' | 'xl' | 'xxl';
```

Comment:
Does it make sense to provide `xs` too?

Reply:
Ok, just saw: https://opensource.adobe.com/spectrum-css/thumbnail.html - there isn't xs provided. So it should be good here.